### DISCUSSION TITLE
Structuring

### DISCUSSION THREAD

**Discussion by wjwat** [wjwat/lhtlhtp.com#3] [Q&A]
How will we structure the site? Is it a vital cliffs notes? Is it something as verbose and detailed as learnhowtoprogram.com? How can we make this site as effective as possible in helping people stepping into Epicodus, and ourselves, with their transition into new roles in life?

**Comment by probablynotryan** [comment]
Here's some things that I wish existed back then: 
- Summary of what's being taught for that section, bolding the vital concepts
- Video demos of hot topics (ie: forloops that Erik did) (hell, i'd volunteer for that)
- Notes from upper cohorts (maybe represented by postits on the page) "UpperCohortBob adds: lorem ipsum etc"
- Specific ways to handle when dependency versions don't line up and you're eating Ben and Jerry's and being sad when it doesn't
- Shortcuts, ways to utilize ways VS code / terminal that can make your life easier (.container (tab)) (arrow up / arrow down in terminal) etc etc
- A podcast with will and ryan called "rill talk" discussing what we already talk about damn near ever day. (Mostly code)

I really enjoy the direction with this in it being an open discussion, but I think something that's also important is to set treats (such as the aforementioned)  at the front door so people brand new to all of these can be a little less intimidated.

**Comment by wjwat** [comment]
Great points. Do you think it would be worth it to record our own videos or just link to freely available ones on places like YouTube? The thing I'm struggling with deciding at the moment is how to best give people the info they want/need at the start of a section. It's been so long since we did `Intro to Programming` that I'm not sure what would be most useful in a section like that right now.

**Reply by probablynotryan** [reply]
Maybe the youtube part may have to be shuffled to the back of the priorities until we get some basic stuff up. I recently tossed up what I thought were the important parts of "just HTML" (tags, indentation, a couple pointers, etc). Don't know if we want to keep it as the "intro" page, but it's something a little more directed than what I originally had before.
